README.md

```diff
@@ -120,6 +140,17 @@ fields:
    - logo: "<http://schema.org/logo>"
 ```
 
+### Debug configurations
+
+#### websockets
+This parameter allows you to turn off websockets when you want explicit polling every 5 seconds.
+The `interval` option from the Google Sheet configuration changes this value.
+
+example:
+```yaml
+debug:
```

Comment:
Do we need `debug` here? Can't we just have `websockets`? Why should it be under `debug`?

Reply:
I was using it for debug testing, but I guess I can be 'main' functionality for users who want some more control of the polling system. I'll change it.